### ISSUE TITLE
Latest Debian Image ca-certificates Package Out of Date

### ISSUE THREAD

**Issue by daviddyball** [kubernetes/kops#8153]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**

`Version 1.15.0 (git-9992b4055)`

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**

`Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.9", GitCommit:"500f5aba80d71253cc01ac6a8622b8377f4a7ef9", GitTreeState:"clean", BuildDate:"2019-11-13T11:13:04Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}`

**3. What cloud provider are you using?**

AWS

**4. What commands did you run?  What is the simplest way to reproduce this issue?**

Used latest Debian AMI: `383156758163/k8s-1.15-debian-stretch-amd64-hvm-ebs-2019-09-26`

**5. What happened after the commands executed?**

Running services/pods that use `hostPath` volumes to get the `ca-certificates` (for example `kiam` or `kube2iam`) fail to authenticate with AWS.

Example error for a `kiam-server` container:
```
Post https://sts.amazonaws.com/: x509: certificate signed by unknown authority
```

**6. What did you expect to happen?**

The `ca-certificates` packages should be up-to-date enough to support validating SSL connections to AWS APIs.

**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**

N/A (it's just the AMI at fault)

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

N/A

**9. Anything else do we need to know?**

We're not currently running `aws-authenticator` for K8s, but it is possible that outdated `ca-certificates` packages would cause issues there too.

**Comment by gfdusc** [comment]
Hello,
you can use an AMI ID from https://wiki.debian.org/Cloud/AmazonEC2Image/Buster or generate your images using https://salsa.debian.org/cloud-team/debian-cloud-images, upload it into your account and then reference the image name in your instancegroup config.

**Comment by daviddyball** [comment]
Unfortunately we've already moved our clusters over to CoreOS to get around this issue and I don't have a cluster to hand to run any tests with the images suggested.

If this _is_ fixed in later images then it'd be ideal to get it set as the recommended AMI for users in the AMI manifests within kops so they don't hit this issue. No doubt any users using the recommended `debian` images [here](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md) will very quickly hit this issue on AWS.

**Comment by olemarkus** [comment]
Buster won't work with current k8s because of issues with nftables. This may have been solved in k8s 1.17, but we are a bit far away from that.
